src/libraries/System.Drawing.Common/src/System/Drawing/Imaging/Encoder.cs

```diff
@@ -16,6 +16,21 @@ public sealed class Encoder
         public static readonly Encoder LuminanceTable = new Encoder(new Guid(unchecked((int)0xedb33bce), unchecked((short)0x0266), unchecked((short)0x4a77), new byte[] { 0xb9, 0x04, 0x27, 0x21, 0x60, 0x99, 0xe7, 0x17 }));
         public static readonly Encoder ChrominanceTable = new Encoder(new Guid(unchecked((int)0xf2e455dc), unchecked((short)0x09b3), unchecked((short)0x4316), new byte[] { 0x82, 0x60, 0x67, 0x6a, 0xda, 0x32, 0x48, 0x1c }));
         public static readonly Encoder SaveFlag = new Encoder(new Guid(unchecked((int)0x292266fc), unchecked((short)0xac40), unchecked((short)0x47bf), new byte[] { 0x8c, 0xfc, 0xa8, 0x5b, 0x89, 0xa6, 0x55, 0xde }));
+        
+        /// <summary>
+        ///	Gets an Encoder object that is initialized with the globally unique identifier for the color space category.
```

Comment:
Two requests: remove the "Gets" at the beginning, which we normally only use for properties with a getter. And add a xref link to the Encoder type. Like this:

```suggestion
        ///	An <xref:System.Drawing.Imaging.Encoder> object that is initialized with the globally unique identifier for the color space category.
```